python/mxnet/ndarray/contrib.py

```diff
@@ -28,7 +29,7 @@
 except ImportError:
     pass
 
-__all__ = ["rand_zipfian", "foreach", "while_loop", "cond"]
+__all__ = ["rand_zipfian", "foreach", "while_loop", "cond", "isinf", "isfinite"]
```

Comment:
Could you also update https://github.com/apache/incubator-mxnet/blob/master/docs/api/python/ndarray/contrib.md#contrib ?

Reply:
Done